src/types.ts

```diff
@@ -308,6 +308,8 @@ export interface LNURLWithdrawServiceResponse {
   callback: string; // The URL which LN SERVICE would accept a withdrawal Lightning invoice as query parameter
   k1: string; // Random or non-random string to identify the user's LN WALLET when using the callback URL
   defaultDescription: string; // A default withdrawal invoice description
+  balanceCheck?: string;
```

Comment:
Not sure about this but jef has these

Reply:
check out the lnurl specs. there are more options that the service can support. 
but we currently don't support the balance check option.

same as paylink I think.

but good to have them in the type anyway.

Reply:
I looked at this: https://github.com/fiatjaf/lnurl-rfc/blob/luds/03.md